### PR TITLE
Turning a wheelchair into an assembly no longer makes it lose its motorization and such

### LINKED ISSUE #31118: You can neuter a syndicate wheelchair for the price of two sodas
1. Get gun barrel craft from welding together 2 wirecutted soda cans (makeshift weapons)
2. Attach them to not any wheelchair, but our special red syndicate wheelchair
3. Take the assembly off, and notice your chair has become good and slow again

### PR DESCRIPTION
**Pull request by DeityLink**
Fixes #31118

:cl:
* bugfix: Turning a wheelchair into an assembly no longer makes it lose its motorization and such. (killette3)

### CODE REVIEW

File: code/game/objects/structures/vehicles/wheelchair.dm [hunk 1]
```diff
@@ -28,6 +28,7 @@
 	if(istype(W, /obj/item/weapon/gun_barrel))
 		to_chat(user, "You place \the [W] on \the [src].")
 		var/obj/structure/bed/chair/vehicle/wheelchair/wheelchair_assembly/I = new (get_turf(src.loc))
+		I.base_wheelchair = type
```
Comment (DeityLink) [inline code/game/objects/structures/vehicles/wheelchair.dm]:
```suggestion
		I.base_wheelchair_type= type
```

**Comment by DamianX** [comment]
Recharge/create a brand new power cell with this one simple trick?

**Comment by DeityLink** [comment]
oh fuck it uses power cells? fine I'll just pocket dimension the base wheelchair then.

**Comment by DeityLink** [comment]
There, should be good

**Comment by DamianX** [comment]
What happens if someone is sitting on the wheelchair as it's transformed?

**Comment by DeityLink** [comment]
Oh god why can you do that. I'll add a check and message so it fails if someone's buckled in

**Comment by DeityLink** [comment]
there it'll unlock all atoms

**Pull request merged**